DevCycle/Networking/DevCycleService.swift

```diff
@@ -200,7 +167,7 @@ class DevCycleService: DevCycleServiceProtocol {
                       let responseDataJson = try? JSONSerialization.jsonObject(with: responseData, options: .fragmentsAllowed) as? [String:Any]
                 else {
                     Log.error("Unable to parse API Response", tags: ["api"])
-                    completion?((nil, nil, APIError.NoResponse))
+                    completion((nil, nil, APIError.NoResponse))
```

Comment:
are the double brackets needed? `(( ))`?

Reply:
yes the  completion handler passes it as a Config object for ease of use everywhere else